symengine/lib/symengine_wrapper.pyx

```diff
@@ -825,6 +825,9 @@ cdef class Basic(object):
     def is_Matrix(self):
         return False
 
+    def copy(self):
+        return self.__class__(*self.args)
```

Comment:
This directly follows from `SymPy`'s implementation of `Basic`. Should I remove it then?